homeassistant/components/vegehub/coordinator.py

```diff
@@ -0,0 +1,52 @@
+"""Coordinator for the Vegetronix VegeHub."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from vegehub import VegeHub, update_data_to_ha_dict
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+type VegeHubConfigEntry = ConfigEntry[VegeHub]
+
+
+class VegeHubCoordinator(DataUpdateCoordinator[dict[str, Any]]):
+    """The DataUpdateCoordinator for VegeHub."""
+
+    config_entry: VegeHubConfigEntry
+    vegehub: VegeHub
```

Comment:
```suggestion
```

Reply:
no I mean that the type can be inferred as this doesn't remove the declaration, it just removes the type hint :)